site/static/js/main.js

```diff
@@ -8,8 +8,17 @@ $(document).ready(function(){
         $(this).toggleClass('fa-plus-square-o fa-minus-square-o');
     });
 
+    if(window.name){
+        $(".notification-bar").addClass("closed");
+        $("header").removeClass("with-notification-bar");
+    }
+
     $('.notification-bar__close').on('click', function(){
-       $(".notification-bar").addClass("closed");
-       $("header").removeClass("with-notification-bar");
+        if(!window.name){
+            $(".notification-bar").addClass("closed");
+            $("header").removeClass("with-notification-bar");
+        }
+        window.name = true;
```

Comment:
It seems odd to use `window.name` for this purpose. Could you please elaborate why you choose this approach vs using a Cookie?

Reply:
No specific reason as such...we can go with session cookie as well